qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -49,6 +49,8 @@ body {
     * ----------------------------------------------------------------------- */
     /* Use gray for the top-level links to be more muted. */
     --color-sidebar-link-text--top-level: var(--color-sidebar-link-text);
+    /* Turn off gradient for hover. */
+    --color-sidebar-item-background--hover: #efeff4;
```

Comment:
that was quick, thanks @Eric-Arellano 

🚀

Reply:
Heh I really want to get out Furo 1.0 by the end of the month :D Thanks for the suggestion!